pkg/ingester/ingester_test.go

```diff
@@ -7091,6 +7091,51 @@ func TestIngesterActiveSeries(t *testing.T) {
 
 				// Check tracked Prometheus metrics
 				require.NoError(t, testutil.GatherAndCompare(gatherer, strings.NewReader(expectedMetrics), metricNames...))
+
+				series, err := listActiveSeries(context.Background(), ingester.getTSDB(userID), [][]*labels.Matcher{{labels.MustNewMatcher(labels.MatchEqual, "team", "a")}, {labels.MustNewMatcher(labels.MatchEqual, "team", "b")}})
+				require.NoError(t, err)
```

Comment:
I see this test passes, but I'm not sure why. Shouldn't we receive an error if active series are disabled?

Reply:
Yep, it passes because in the current code `userTSDB.activeSeries` is never nil, even when active series are disabled.

Reply:
Ok I see now. So if the active series tracker is disabled, then this endpoint returns an empty list. And after checking further, it looks like the other endpoints work similarly. Sounds good.